contracts/LifMarketMaker.sol

```diff
@@ -298,4 +298,23 @@ contract LifMarketMaker is Ownable {
     // foundationAddr.transfer(this.balance);
   }
 
+  function pause() onlyOwner whenNotPaused {
+    paused = true;
+    blockPaused = block.number;
+    Pause();
+  }
+
+  function unpause() onlyOwner whenPaused {
+    uint256 blocksPaused = block.number.sub(blockPaused);
+    uint256 periodPaused = blockPaused.sub(startBlock).div(blocksPerPeriod);
+    marketMakerPeriods[periodPaused].endBlock = marketMakerPeriods[periodPaused].endBlock.add(blocksPaused);
```

Comment:
Can we avoid updating all the blocks in unpause? Maybe keeping a general startBlock and then having every period's start and end blocks to be relative to that `startBlock.plus(blocksPaused)`?

Reply:
For that we need to remove startBlock and endBlock form the periods structure, since we wont need it anymore. I think its a better solution, but we can leave it for the next PR.

Reply:
I think we should do it in this PR, it'd be the best if we can do that:

- Initial commit with the change to having startBlock + relative blocks in the periods (as you said, probably removing start and end block from periods altogether)
- Second commit adding pausable

Reply:
ok, will do